scratch/sim_correlator.py

```diff
@@ -139,7 +133,7 @@ def generate_digitisers(args: argparse.Namespace, config: dict) -> list[str]:
                     "band": args.band[:1],
                     "adc_sample_rate": args.adc_sample_rate,
                     "centre_frequency": args.centre_frequency,
-                    "antenna": f"m{number}",
+                    "antenna": f"m{number}, 0:0:0, 0:0:0, 0, 0",
```

Comment:
Also, I didn't realise this was all that was needed for the dumm descriptions.

Reply:
Well, it'll allow SDP to run. But SDP does RFI flagging for longer (>1km) versus shorter baselines, so for copy-mk it needs more accurate information.